src/image_tools/args.py

```diff
@@ -90,6 +90,12 @@ def bake_args() -> Namespace:
         type=check_build_arg,
     )
 
+    parser.add_argument(
+        "--target-containerfile",
```

Comment:
Why is it called --target-containerfile and not just --containerfile?

Reply:
It is called like take, because the JSON key in the bake file uses that name. See: https://docs.docker.com/build/bake/reference/#targetdockerfile

Reply:
Alright, works for me, thanks!